### PR TITLE
Replace 'enabledMethods' array with 'method' string.

### PR DESCRIPTION
**Pull request by rsolomakhin**
The following tasks have been completed:
 * [x] [web platform tests](https://github.com/w3c/web-platform-tests/pull/10180)

***
<a href="https://pr-preview.s3.amazonaws.com/rsolomakhin/payment-handler/pull/276.html" title="Last updated on Apr 3, 2018, 3:09 PM GMT (3281123)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/276/4a98c94...rsolomakhin:3281123.html" title="Last updated on Apr 3, 2018, 3:09 PM GMT (3281123)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -135,8 +135,8 @@ <h2>
         <li>During service worker registration, the <a>PaymentManager</a> is
         used to set:
           <ul>
-            <li>A list of <a data-lt="PaymentInstrument.enabledMethods">enabled
-            payment methods</a>.
+            <li>A list of <a data-lt="PaymentInstrument.method">enabled payment
+            methods</a>.
```
Comment (ianbjacobs) [inline index.html]: @rsolomakhin,

On the one hand, I think we do want to talk about payment methods here. On the other hand, the WebIDL says "PaymentInstruments". I'm not recommending a change at this point; I just want to point this out.

**Review by ianbjacobs — APPROVED** [review]

**Review by marcoscaceres — APPROVED** [review]
minor nit, but could this be two commits? One for the `"` fixes, and another for the actual changes. If it's too much of a pain, don't worry.

File: index.html (hunk 1 above)
Comment (ianbjacobs) [reply index.html]: Here's a suggestion for re-writing bullet 3 of this list (in section 2 of the spec):
<pre>
<li>During service worker registration, the <a>PaymentManager</a> is
used to set a list of <a data-lt="PaymentInstrument.PaymentInstrument">payment instruments</a>. Each <a data-lt="PaymentInstrument.PaymentInstrument">payment instrument</a> provides data to the user agent to improve the user experience of selecting payment credentials:
 <ul>
   <li>The <a data-lt="PaymentInstrument.method">method</a> and optional <a data-lt="PaymentInstrument.capabilities">capabilities</a> inform the user agent decision whether to display this instrument as a candidate for payment.</li>
   <li>When the instrument matches what the payee accepts, the user agent displays the <a data-lt="PaymentInstrument.name">Name</a> and <a data-lt="PaymentInstrument.icon">icon</a>. These provide hints about payment credentials that the user agent will return in the <a>PaymentHandlerResponse</a> if the user selects this instrument.</li>
 </ul>
</li>
</pre>

Comment (ianbjacobs) [reply index.html]: (Look at the source of that comment for the markup)

**Comment by rsolomakhin** [comment]
> could this be two commits? One for the " fixes, and another for the actual changes.

Done: https://github.com/w3c/payment-handler/pull/278

File: index.html (hunk 1 above)
Comment (rsolomakhin) [reply index.html]: Done.

**Pull request merged**